### PR TITLE
Backoffice series search

### PR DESCRIPTION
**Pull request by kpsherva**
* removes labels from the facets
* reorganise series search page
* add facets to series search

Screenshot 2019-12-11 at 15 52 08
Screenshot 2019-12-11 at 15 52 27

BEFORE:
Screenshot 2019-12-11 at 18 38 53

### CODE REVIEW

**Comment by ntarocco** [comment]
Comments on the screenshots:
* when you display `languages: EN, ...` in each row, does it mean that this series has a language relation with another series in EN language?  Or is it the language of the series?
* since `relations` and `mode_of_issuance` are hardcoded/config, you can safely created the "translation" in the `config.js` to display a readable text in facets and each row

**Review by ntarocco — APPROVED** [review]

File: ui/src/components/SearchControls/components/SearchAggregations/SearchAggregationsCards.js [hunk 1]
```diff
@@ -37,7 +28,7 @@ export default class SearchAggregationsCards extends Component {
           </List.Content>
           <List.Content>
             <Checkbox
-              label={label}
+              label={`${bucket.key}`}
```
Comment (equadon) [inline ui/src/components/SearchControls/components/SearchAggregations/SearchAggregationsCards.js:31]:
```suggestion
              label={bucket.key}
```
The bucket key is already a string, right?

File: ui/src/config/invenioConfig.js [hunk 2]
```diff
@@ -132,5 +132,6 @@ export const invenioConfig = {
 };
 
 export function getDisplayVal(configField, value) {
-  return _get(invenioConfig, configField).find(entry => entry.value === value);
+  return _get(invenioConfig, configField).find(entry => entry.value === value)
```
Comment (equadon) [inline ui/src/config/invenioConfig.js]: Perhaps raise an exception with a meaningful message if no entry was found?

File: ui/src/pages/backoffice/components/icons/Icons.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import {Icon} from "semantic-ui-react";
+
+export class DocumentIcon extends Component{
```
Comment (equadon) [inline ui/src/pages/backoffice/components/icons/Icons.js]: :+1:

**Review by equadon — APPROVED** [review]
Looks good!

The reason it's showing ISSN for multipart monographs now is because our test data has it, right?

Some series titles can get quite long if I recall correctly, how are we gonna handle those?

**Pull request merged**